src/components/product-switcher/index.tsx

```diff
@@ -213,9 +213,7 @@ const ProductSwitcher: React.FC = () => {
       >
         <span className={s.switcherOptionContainer}>
           <ProductIcon
-            className={classNames({
-              [s.vaultProductIcon]: currentProduct?.slug === 'vault',
-            })}
+            className={s[`${currentProduct?.slug}ProductIcon`]}
```

Comment:
Missed this in an earlier PR, fixes an issue where the HCP icon is display black-on-dark-background instead of the intended white-on-dark-background.